### ISSUE TITLE
Add more elements to att.docStatus

### ISSUE THREAD

**Issue by martindholmes** [TEIC/TEI#2013]
Following this discussion on TEI-L:

http://tei-l.970651.n3.nabble.com/Encoding-status-of-items-rather-than-documents-td4032773.html

this ticket suggests that there is a need to be able to specify the status (draft, proofed, published etc.) not just of documents but of a variety of different elements which don't constitute documents in themselves. These are elements appearing in resource lists such as listPerson/person, listPlace/place, listBibl/bibl, list/item, listObject/object, and similar "entity records". 

The definition of att.docStatus specifically mentions that it applies to documents, but it could be re-defined so that its name is interpreted as "document the status of...". Alternatively, a new attribute class such as att.resourceStatus or att.fragmentStatus could be created.

**Comment by laurentromary** [comment]
Thanks a lot Martin for pushing this.
I am using the @status myself at various places and list a few sample of usages and values here:
TEI: general validation status of the document
listPlace: to indicate the applicability of a certain legal document to a certain place
•	E.g.: status="original" – the places like they were originally set at the beginning of the procedures
•	E.g.: status=”extended”, when it has been extended
biblStruct: to indicate whether the bibliographical reference has been checked (after extraction from a source)
•	E.g.: extracted
textClass, classCode: to indicate how far the codes are agreed to classify a document:
•	E.g.: unreviewed, confirm, riseHand
title: to indicate the degree of stability/validity of a title given to a legal document (since this may change along the legal procedure it undergoes)
•	E.g. undetermined
idno
•	E.g. extracted
language (status of the language with regards a multilingual procedure; written procedural language, oral proceedings)
address: idem (extraction)
event: to represent the legal status of an event (border line)
For <when>, I have introduced a specific class att.elementStatus with @status that can have two values: given or unknown for when the underlying database is given me no value.

**Comment by ebeshero** [comment]
Indeed, I have an immediate application for this from the Digital Mitford project's prosopography lists, as these are often very much in progress and some are just "stubs" in need of development. Reviewing the emails on the TEI list, the use of @resp is often what we go with to tell who's been through an entry and get a sense of its status. I like being able to formalize the status of an entry (as well as a document) as @martindholmes proposes here, though I am tripped up a little by the element name `docStatus` and wish it were something more generally applicable. I suppose modifying the definition can make it work, and perhaps we think of the question of what a "document" is, really! Can we think of a "document" as an entry in a prosopography list, the status of which can be classified?

**Comment by martindholmes** [comment]
I don't really know whether documents actually exist as distinct entities any more, in most TEI collections. Files certainly exist, but what we think of as a "document" is so interlinked and interdependent with other resources that it's almost a meaningless concept. On balance, I think `@status` in `att.status` would make more sense, with its membership constituting anything that is arguably a discrete record-like thing, including a document; when applied to `<revisionDesc>` it could be defined as applying to the whole "document", but elsewhere just to the element on which it appears.

**Comment by raffazizzi** [comment]
VF2F is considering a global `@status` attribute, but that will require adjusting other classes:
* `att.transcriptional`
* `att.docStatus`

Assignees will provide an overview of what would need to be changed, which will likely include deprecations, to facilitate further discussion.

**Comment by hcayless** [comment]
`@status` is used by 
1. [att.docStatus](https://tei-c.org/Vault/P5/4.1.0/doc/tei-p5-doc/en/html/ref-att.docStatus.html) 
2. [att.transcriptional](https://tei-c.org/Vault/P5/4.1.0/doc/tei-p5-doc/en/html/ref-att.transcriptional.html)
3. [availability](https://tei-c.org/Vault/P5/4.1.0/doc/tei-p5-doc/en/html/ref-availability.html)
4. [correction](https://tei-c.org/Vault/P5/4.1.0/doc/tei-p5-doc/en/html/ref-correction.html)

If we wish to make `@status` in the sense of 1) available globally, then we would need to rename att.docStatus to something like att.global.status and redefine it so that it can be understood to apply to elements as well as documents.

Att.transcriptional affects only a few elements, so we could make the new model class not-quite-global and deprecate/rename `@status` on att.transcriptional. `@status` on availability is sane but reductive and probably not very useful, so could be deprecated. `@status` on correction is a bit mad and unlikely to be useful, so could (should?) be deprecated.

Alternatively, as @martindholmes suggests, we could define a new attribute class that applies to elements/document fragments and define it accordingly. This would avoid our having to torture the semantics of the attribute, particularly when applied to `<change status="">`, where it should clearly apply to the document rather than the change element itself.

Another option would be to change nothing, add a new attribute `@edStatus` (or something) in a new attribute class and make it global.

I suggest removing the `defaultVal` from att.docStatus whether we change it or not.

**Comment by martindholmes** [comment]
@hcayless Couldn't you just override `@status` where it currently appears in att.transcriptional, availability and correction? Its gloss/desc can be overridden just as well as its possible values. I'm not quite sure what the implications of doing that are; we usually think of an attribute override as a way to further constrain or fine-tune its valList, rather than redefine the attribute.

**Comment by hcayless** [comment]
@martindholmes I don't really like the idea of using the override mechanism to completely redefine an attribute. But maybe in conjunction with deprecation it would be ok...

**Comment by raffazizzi** [comment]
I think we have @hcayless's proposal to discuss here before making a final decision and proceeding. Changing this to Needs Discussion.

**Comment by joeytakeda** [comment]
@martindholmes 's original Nabble link is broken, so here is a link to the PSU listServ: https://lists.psu.edu/cgi-bin/wa?A2=TEI-L;6dc3e946.2007&FT=&P=R157&H=&S=

**Comment by ebeshero** [comment]
Council F2F discussion: Go to implement. We can add a new attribute `@edStatus` as a member of `att.global.change` with `tei.enumerated` to be glossed as "editorial status".

Aside: We will want to reduce the complexity of this status in P6
